packages/app/src/domain/escalation-level/scoreboard/logic/sort-options.tsx

```diff
@@ -0,0 +1,78 @@
+import { VrScoreboardData } from './scoreboard-types';
+
+export const scoreboardSortOptions = {
+  location_a_to_z: (a: VrScoreboardData, b: VrScoreboardData) => {
+    if (a.safetyRegionName < b.safetyRegionName) return -1;
+    if (b.safetyRegionName < a.safetyRegionName) return 1;
+    return 0;
+  },
+  location_z_to_a: (a: VrScoreboardData, b: VrScoreboardData) => {
+    if (a.safetyRegionName > b.safetyRegionName) return -1;
+    if (b.safetyRegionName > a.safetyRegionName) return 1;
+    return 0;
+  },
+  positively_tested_high_to_low: (a: VrScoreboardData, b: VrScoreboardData) => {
+    if (
+      (a.data.positive_tested_per_100k as number) >
```

Comment:
you can't assume this is always a valid number, better to write it like this:
```suggestion
      (a.data.positive_tested_per_100k ?? -Infinity) >
```

Reply:
Same goes for the rest of the casts of course